src/IntelliTect.Coalesce/TypeDefinition/PropertyViewModel.cs

```diff
@@ -349,9 +349,13 @@ public bool IsSearchable(ClassViewModel rootModel)
         /// <summary>
         /// Returns the Linq.Dynamic-interpretable string representation of the method invocation that will perform search on the property.
         /// </summary>
-        public string SearchMethodCall => SearchMethod == SearchAttribute.SearchMethods.Contains
-            ? @"ToLower().IndexOf((""{0}"").ToLower()) >= 0"
-            : @"ToLower().StartsWith((""{0}"").ToLower())";
+        public string SearchMethodCall => SearchMethod switch {
+            SearchAttribute.SearchMethods.Contains => @"ToLower().IndexOf((""{0}"").ToLower()) >= 0",
+            SearchAttribute.SearchMethods.BeginsWith => @"ToLower().StartsWith((""{0}"").ToLower())",
+            SearchAttribute.SearchMethods.Equals => @"ToLower().Equals((""{0}"").ToLower())",
```

Comment:
Do you think we can get away from the ToLowers with .Equals(string, StringComparison.InvariantCultureIgnoreCase)? Seems like EF should handle that. Maybe faster?

Reply:
No, because EF doesn't actually handle it (https://github.com/dotnet/efcore/issues/1222, https://github.com/dotnet/efcore/issues/18881), and Linq.Dynamic also can't figure out how to parse the enum value.